### PR TITLE
update /cloud/plans-and-pricing with new pricing includes

### LINKED ISSUE #2236: Autopilot messaging to be replaced in the footer.
Autopilot messaging is  still in footer. This should be removed and replaced with a default message.
Please see default messages in this doc: https://docs.google.com/document/d/1kq0SPv-PAIfL9nHGxgNDpHhr5OplE2iNaZZiqG4RT48/edit 

---
*Reported from: https://www.ubuntu.com/cloud/plans-and-pricing*

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

* remove autopilot from contextual footer
* updated with new includes
* deleted several old includes
* made the OpenStack table col-10 from col-12
* fixed some bad CTAs and google events

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [cloud p&p page](http://0.0.0.0:8001/cloud/plans-and-pricing) and [support p&p page](http://0.0.0.0:8001/support/plans-and-pricing)
- compare to the [cloud p&p copy doc](https://docs.google.com/document/d/1yZ7wClcmxGCqMyyyBJLO66Z3YlpT-4soWLFmJ6uXeMM/edit#heading=h.tp2i0jhbkhwa) and [support p&p copy doc](https://docs.google.com/document/d/1R7FRZbfx_5YHFeZUD93HgT8RW4OCbbK4blOulB4D6EE/edit#)

## Issue / Card

Fixes #2236

### CODE REVIEW

**Comment by frankiegranato** [comment]
Wow - I’m super impressed at how fast you did this.

A couple of small snags to be fixed:

The “Buy Ubuntu Advantage ›” cta goes to a contact form and it should be https://buy.ubuntu.com/

The “Schedule a BootStack demo ›” cta should go to https://www.ubuntu.com/cloud/contact-us?product=cloud-managed-cloud

The contact cta for “...Ceph and Swift Storage” section should go to: https://www.ubuntu.com/cloud/contact-us?product=cloud-storage (this is a error that I must have missed on the support/P&P too so I’m filing a bug now)

**Comment by pmahnke** [comment]
@frankiegranato I fixed the cta's including the /support one.

File: templates/cloud/plans-and-pricing.html [hunk 1]
```diff
@@ -13,142 +13,104 @@
...
 
-{% include "shared/_pricing_landscape_on-premises.html" %}
-
 <section class="p-strip is-bordered" id="bootstack-fully-managed-cloud">
   <div class="row">
     <div class="col-9">
       <h2>BootStack fully managed OpenStack Cloud</h2>
       <p>Let us build, operate and support your OpenStack Cloud. Available as a cost per server per day or&nbsp;per&nbsp;VM per hour.</p>
-      <p><a href="/cloud/contact-us?product=managed-cloud">Schedule a BootStack demo&nbsp;&rsaquo;</a></p>
+      <p><a href="/cloud/contact-us?product=cloud-managed-cloud">Schedule a BootStack demo&nbsp;&rsaquo;</a></p>
```
Comment (grahambancroft) [inline templates/cloud/plans-and-pricing.html]: The link is different in the copy doc

File: templates/cloud/plans-and-pricing.html [hunk 2]
```diff
@@ -181,6 +143,31 @@ <h3 class="p-heading--five">Add-ons</h3>
...
+  <div class="row">
+    <div class="col-10">
+      {% include "shared/pricing/_storage-support.html" %}
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="col-12">
+      <div>
+        <p><a href="/cloud/contact-us?product=cloud-storage" class="p-button--neutral" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Contact us', 'eventLabel' : 'Contact us - cloud storage CTA' : undefined });">Contact us</a></p>
```
Comment (grahambancroft) [inline templates/cloud/plans-and-pricing.html]: Different link in copy doc

File: templates/cloud/plans-and-pricing.html [hunk 3]
```diff
@@ -13,142 +13,104 @@
...
-  <div class="row">
-    <div class="col-8">
-      <h2>Cloud tools</h2>
-      <p>Ubuntu provides tools to help you create, scale and manage your cloud.</p>
+<section class="p-strip--light is-deep is-bordered">
+  <div class="row u-equal-height u-padding-bottom">
+    <div class="col-4 p-card">
+      <h3>OpenStack</h3>
+      <div class="p-card__footer">
+        <p class="p-heading--five">From $4 per day <span style="font-size: 1rem;">*</span></p>
```
[the remark this answers is not in the document]
Comment (pmahnke) [reply templates/cloud/plans-and-pricing.html]: done

File: templates/cloud/plans-and-pricing.html [hunk 4]
```diff
@@ -13,142 +13,104 @@
...
+        <a href="#openstack">Learn more&nbsp;&rsaquo;</a>
+      </div>
+    </div>
+    <div class="col-4 p-card">
+      <h3>Consulting packages</h3>
+      <div class="p-card__footer">
+        <p class="p-heading--five">From $150 per year <span style="font-size: 1rem;">*</span></p>
+      </div>
+      <div class="p-card__footer">
+        <a href="#consulting">Learn more&nbsp;&rsaquo;</a>
```
[the remark this answers is not in the document]
Comment (pmahnke) [reply templates/cloud/plans-and-pricing.html]: good one... done.

File: templates/cloud/plans-and-pricing.html [hunk 5]
```diff
@@ -13,142 +13,104 @@
...
+      </div>
+    </div>
+    <div class="col-4 p-card">
+      <h3>Storage</h3>
+      <div class="p-card__footer">
+        <p class="p-heading--five">From $0.01 per month <span style="font-size: 1rem;">*</span></p>
```
[the remark this answers is not in the document]
Comment (pmahnke) [reply templates/cloud/plans-and-pricing.html]: done

[the remark this answers is not in the document]
Comment (pmahnke) [reply templates/cloud/plans-and-pricing.html]: done

File: templates/cloud/plans-and-pricing.html [hunk 6]
```diff
@@ -13,142 +13,104 @@
...
+      </div>
+      <div class="p-card__footer">
+        <a href="#storage">Learn more&nbsp;&rsaquo;</a>
```
[the remark this answers is not in the document]
Comment (pmahnke) [reply templates/cloud/plans-and-pricing.html]: good one... done

File: templates/cloud/plans-and-pricing.html (hunk 1 above)
Comment (pmahnke) [reply templates/cloud/plans-and-pricing.html]: this is correct, I updated the copy doc

File: templates/cloud/plans-and-pricing.html (hunk 2 above)
Comment (pmahnke) [reply templates/cloud/plans-and-pricing.html]: this is correct, I updated the copy doc

**Comment by pmahnke** [comment]
@grahambancroft ready for re-review

**Pull request merged**